exporter/prometheusremotewriteexporter/config.go

```diff
@@ -104,7 +114,7 @@ func (cfg *Config) Validate() error {
 	}
 	if cfg.CreatedMetric == nil {
 		cfg.CreatedMetric = &CreatedMetric{
-			Enabled: false,
+			Enabled: exportCreatedMetricGate.IsEnabled(),
```

Comment:
```suggestion
			Enabled: false,
```

Let's go back to false!